Eatery/Controllers/Eateries/Campus/ScrollableViewController.swift

```diff
@@ -0,0 +1,374 @@
+        headerContainer.backgroundColor = .white
+        headerContainer.directionalLayoutMargins = NSDirectionalEdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16)
+
+        headerContainer.snp.makeConstraints { make in
+            make.height.equalTo(58)
+        }
+
+        let titleLabel = UILabel()
+        titleLabel.text = "Menu"
+        titleLabel.textColor = .black
+        titleLabel.font = .boldSystemFont(ofSize: 26)
+        titleLabel.preservesSuperviewLayoutMargins = true
+        headerContainer.addSubview(titleLabel)
+
+        filterButton = ExtendedFilterButton(frame: .zero, inactiveColor: .systemGray, activeColor: .eateryBlue)
+        filterButton.addTarget(self, action: #selector(filterButtonPressed), for: .touchUpInside)
+        filterButton.preservesSuperviewLayoutMargins = true
+        headerContainer.addSubview(filterButton)
+
+        filterLabel = UILabel()
+        filterLabel.textColor = .eateryBlue
+        filterLabel.font = .boldSystemFont(ofSize: 11)
+        filterLabel.text = ""
+        filterLabel.preservesSuperviewLayoutMargins = true
+        headerContainer.addSubview(filterLabel)
+
+        separatorView = SeparatorView(insets: UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0))
+        separatorView.tintColor = .clear
+        separatorView.isHidden = true
+        headerContainer.addSubview(separatorView)
+
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(tabBarPadding)
+            make.centerY.equalToSuperview()
+            make.bottom.equalToSuperview()
+        }
+
+        filterButton.snp.makeConstraints { make in
+            make.centerY.equalTo(titleLabel.snp.centerY)
+            make.height.equalTo(titleLabel.snp.height).multipliedBy(0.3)
+            make.trailing.equalToSuperview().offset(-tabBarPadding)
+            make.width.equalTo(filterButton.snp.height).multipliedBy(1.71)
+        }
+
+        filterLabel.snp.makeConstraints { make in
+            make.centerY.equalTo(titleLabel.snp.centerY)
+            make.trailing.equalTo(filterButton.snp.leading).offset(-6)
+        }
+
+        separatorView.snp.makeConstraints { make in
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+
+        let titles = viewControllers.map {
+            $0.category
+        }
+        tabBar = ScrollableTextTabBarControl(sections: titles, padding: 16)
+        tabBar.directionalLayoutMargins = NSDirectionalEdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16)
+        tabBar.addTarget(self, action: #selector(manualScroll), for: .valueChanged)
+        tabBar.backgroundColor = .white
+        tabBar.layer.zPosition = 2
+
+        tabBar.snp.makeConstraints { make in
+            make.height.equalTo(40)
+        }
+
+        tabBarStack = UIStackView(arrangedSubviews: [headerContainer, tabBar])
+        tabBarStack.axis = .vertical
+    }
+
+    private func createMenuStack() {
+        menuItemsView = UIView()
```

Comment:
May be worthwhile to pull this out into its own class as well

Reply:
Would it be possible for me to create an ExpandedMenus folder within Eatery/Views/Eateries/ ? There are a lot of views in this folder that have to do with Expanded Menus and they're scattered all over the place.  I'm not sure if we could categorize the other views too but I think it would be a solid idea because there's A LOT in this folder.  Let me know what you think

<img width="274" alt="Screen Shot 2021-01-27 at 7 45 17 PM" src="https://user-images.githubusercontent.com/56354603/106073519-3a2a8680-60d8-11eb-825d-de03c19bcb96.png">